integration/commands_freemonitoring_test.go

```diff
@@ -53,3 +54,79 @@ func TestCommandsFreeMonitoringGetFreeMonitoringStatus(t *testing.T) {
 		assert.Equal(t, m[k], item)
 	}
 }
+
+func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
+	t.Parallel()
+	ctx, collection := setupWithOpts(t, &setupOpts{
+		databaseName: "admin",
+	})
+
+	for name, tc := range map[string]struct {
+		command  bson.D
+		expected map[string]any
+		err      *mongo.CommandError
+	}{
+		"SetFreeMonitoring_Enable": {
```

Comment:
```suggestion
		"Enable": {
```

So the full test name will be `TestCommandsFreeMonitoringSetFreeMonitoring/Enable`